helix-loader/src/grammar.rs

```diff
@@ -443,10 +443,42 @@ fn build_tree_sitter_library(
             .arg(header_path)
             .arg("/Od")
             .arg("/utf-8")
-            .arg("/std:c++14")
             .arg("/std:c11");
```

Comment:
Won't we need to move this into the block for c files too?

Reply:
yup that's right I fogot to commit something. I already suspected that the windows test CI did not in fact build the grammars but wanted to investigate further once the source hut DOS if over

Reply:
still gonna clear the caches later just to be sure tough

Reply:
Oh I see, the C++ command is a totally separate command; at first glance, I thought it was adding these options to the same command. This makes sense now.